### PR TITLE
Update tour remind

### PR DESCRIPTION
**Pull request by Volco**
This now allows tour remind to pm all users in the tour

### CODE REVIEW

File: tournaments/index.js [hunk 1]
```diff
@@ -1224,11 +1224,9 @@ let commands = {
 		},
 		remind: function (tournament, user) {
 			let offlineUsers = [];
-			tournament.generator.getAvailableMatches().toString().split(",").forEach(u => {
+			tournament.generator.getAvailableMatches().toString().split(',').forEach(u => {
 				let targetUser = Users.get(u);
-				if (!targetUser) {
-					offlineUsers.push(u);
-				} else if (!targetUser.connected) {
+				if (!targetUser.connected) {
```
Comment (HoeenCoder) [inline tournaments/index.js]: This crashes if targetUser is undefined, which can happen if its run before the tournament starts.

Comment (HoeenCoder) [reply tournaments/index.js]: Also it seems to be occuring even after i start the tournament

Comment (Volco) [reply tournaments/index.js]: didnt crash when i tested it ill take another look

Comment (Volco) [reply tournaments/index.js]: yeah so this is gonna take a while to fix it seems so just closing it

**Comment by HoeenCoder** [comment]
Errored check is my fault, i didn't test a commit earlier.

Tour remind shouldn't work before the tournament has started, and if it should work before the tour starts its: **1.** not pming the users, **2.** coming up with `The following users are currently offline: BracketNotFrozen.`

**Comment by Volco** [comment]
even azures tour remind does the BracketNotFrozen thing but ill look into why it wouldnt be pming users it pmed them all in testing

**Comment by HoeenCoder** [comment]
Possibly check to see if the tour hasn't started? if so cut `BracketNotFrozen` from the offline list? Its a pretty easy patch tbh.

**Pull request merged**